coursedashboards/static/coursedashboards/js/main.js

```diff
@@ -299,14 +299,28 @@ function setup_exposures($container) {
             $(this).parent().hide();
         }
     });
-    $container.find(".show-more").on('click', function () {
-        var $hidden = $(this).closest('.list').find('ul.list-unstyled li:hidden');
-        // show next ten, then hide control
-        $hidden.slice(0,10).show();
+    $container.find(".toggle-show").on('click', function () {
+        var expanded = $(this).attr("expanded");
+        console.log(expanded)
```

Comment:
Can this go as well as the commented lines below?

Reply:
Yeah, thanks for catching that!